### ISSUE TITLE
Can I use opendap?

### ISSUE THREAD

**Issue by kthyng** [intake/intake-stac#99]
Hi! I [see that](https://intake-stac.readthedocs.io/en/latest/_modules/intake_stac/catalog.html?highlight=driver#) the "STAC catalog asset 'type' determines intake driver" but I don't see opendap in the list there, while it is [listed separately](https://intake-xarray.readthedocs.io/en/latest/quickstart.html#choosing-a-driver) for `intake-xarray` as a different driver. Is there a way I can specify "opendap" as the driver, or some other way around this? Thank you!

**Comment by scottyhq** [comment]
Hello @kthyng , yes I suppose that isn't implemented b/c nobody has expressed interest in opendap+STAC that I'm aware of. Out of curiosity, what STAC catalog are you using? If no STAC catalog exists, you can use intake-xarray directly (and we should probably remove mention of opendap and GRIB from this repo's readme.

Or adding a driver would require adding something like `'application/opendap': 'opendap'` to https://github.com/intake/intake-stac/blob/7d89adc1467fd1db8e81cfc49453621fb7483ea6/intake_stac/catalog.py#L18 

And we'd need someone to write tests to ensure things work as expected.

**Comment by kthyng** [comment]
Thanks for your response @scottyhq! I am investigating making a STAC catalog of local assets at my company to then be interacted with via `intake-stac`. I am just starting my investigating — is this not a good approach?

**Comment by kthyng** [comment]
I am willing to work on this and make a PR but wanted to make sure there isn't an obvious reason to avoid this approach.

**Comment by scottyhq** [comment]
@kthyng - i'd recommend adding to this forum thread or opening a new post to try to get some feedback on your approach https://discourse.pangeo.io/t/stac-and-earth-systems-datasets/1472/15

**Comment by kthyng** [comment]
Yes, I have actually been meaning to do that, thanks for the reminder. 

Is this group open to merging a PR to include other drivers in `intake-stac` if I make it and it includes all necessary items?

**Comment by scottyhq** [comment]
> Is this group open to merging a PR to include other drivers in intake-stac if I make it and it includes all necessary items? 

Definitely! Would be great to see an example first using your fork and a STAC catalog that includes opendap endpoints.
